draft-ietf-tls-tls13.md

```diff
@@ -1569,11 +1569,13 @@ cipher_suites
   usual. Values are defined in {{cipher-suites}}.
 
 legacy_compression_methods
-: Versions of TLS before 1.3 supported compression with the list of
-  supported compression methods being sent in this field. For every TLS 1.3
-  ClientHello, this vector MUST contain exactly one byte set to
-  zero, which corresponds to the "null" compression method in
-  prior versions of TLS. If a TLS 1.3 ClientHello is
+: Versions of TLS before 1.3 supported compression of the plaintext prior to
+  encryption, with the list of supported compression methods being sent
+  in this field.  TLS 1.3 does not support this compression functionality
+  but must send a field that would correspond to the "null" compression
```

Comment:
who must?  TLS 1.3 clients MUST send a field that corresponds to the "null" compression method in older versions of TLS (that is, a vector containing a single zero octet: 0x0100).